### PR TITLE
Added Romantic message personality type

### LINKED ISSUE #197: Add new hydrate message personality type: Romantic
## Feature Reasoning

With the introduction of personality types for hydration break messages, it would be nice to add more new personality types. One such personality type to add is: `romantic`.

## Feature Description

The purpose of this feature request is to add a new personality type `romantic` to the currently existing personality types available for the hydrate break messages. The new personality type should be made selectable via the personality type dropdown menu in the hydrate reminder configuration menu.

To start off, adding at least 12 new messages should be good and more can be added later.
An example of a romantic message could be something like: `Roses are red, violets are blue, this hydrate reminder is just for you`

## Possible Alternatives

If not really romantic, any messages that could be considered seductive or deeply loving can be considered OK for this personality type.

## Additional Context

To complete this work, the following two files need to be edited:
- [HydrateReminderPersonalityType.java](https://github.com/jmakhack/hydrate-reminder/blob/master/src/main/java/com/hydratereminder/HydrateReminderPersonalityType.java)
- [HydrateBreakMessageDictionary.java](https://github.com/jmakhack/hydrate-reminder/blob/master/src/main/java/com/hydratereminder/dictionary/HydrateBreakMessageDictionary.java)

Looking into these two files should give a clear indication on how to implement this feature, but please do ask any questions in the comments below if needed. Please also add and/or update unit tests where necessary.

Please refer to [CONTRIBUTING.md](https://github.com/jmakhack/hydrate-reminder/blob/master/CONTRIBUTING.md) for info on how to setup, build, run, and test this project. Feel free to ping @jmakhack in the comments below with any questions if needed. Also, join the [Discord](https://discord.gg/RTgxfFW9mS) to meet and engage with other contributors!

[](https://discord.gg/RTgxfFW9mS)

### PR DESCRIPTION
**Pull request by Zeesky-code**
## Implemented Solution
Added the ROMANTIC personality inside [HydrateReminderPersonalityType.java](https://github.com/jmakhack/hydrate-reminder/blob/master/src/main/java/com/hydratereminder/HydrateReminderPersonalityType.java).

Added the ROMANTIC dictionary into a separate function and then added the option to the switch case inside [HydrateBreakMessageDictionary.java](https://github.com/jmakhack/hydrate-reminder/blob/master/src/main/java/com/hydratereminder/dictionary/HydrateBreakMessageDictionary.java)

## Testing Details
No testing has been done, as the change only adds a new option for the hydration message sent to users. No new functionality was added nor a change in the project functionality was done.
(However, testing will be done if required.)

### CODE REVIEW

**Comment by jmakhack** [comment]
Hi @Zeesky-code, can you leave a comment on the issue #197 so I can assign it to you before somebody else duplicates the work?

File: src/main/java/com/hydratereminder/dictionary/HydrateBreakMessageDictionary.java [hunk 1]
```diff
@@ -81,6 +81,24 @@ public class HydrateBreakMessageDictionary {
                     }}
             );
     
+    private static final List<String> HYDRATE_BREAK_ROMANTIC_TEXT_LIST =
+            Collections.unmodifiableList(
+                new ArrayList<String>(){{
```
Comment (jmakhack) [inline src/main/java/com/hydratereminder/dictionary/HydrateBreakMessageDictionary.java:86]: Can you add a space between the `()` and `{{` here?

File: src/main/java/com/hydratereminder/dictionary/HydrateBreakMessageDictionary.java [hunk 2]
```diff
@@ -102,6 +120,8 @@ public static String getRandomHydrateBreakMessageForPersonality(HydrateReminderP
             case CARING:
                 breakMessage = getRandomBreakMessage(HYDRATE_BREAK_CARING_TEXT_LIST);
                 break;
+            case ROMANTIC:
+                breakMessage =  getRandomBreakMessage(HYDRATE_BREAK_ROMANTIC_TEXT_LIST);
```
Comment (jmakhack) [inline src/main/java/com/hydratereminder/dictionary/HydrateBreakMessageDictionary.java:124]: Please remove the extra whitespace here after the `=`.

**Review by jmakhack — CHANGES_REQUESTED** [review]
Looks like the build is failing after your changes with the following error message:

```
Starting a Gradle Daemon (subsequent builds will be faster)
/Users/runner/work/hydrate-reminder/hydrate-reminder/src/main/java/com/hydratereminder/dictionary/HydrateBreakMessageDictionary.java:100: error: ';' expected

            )
> Task :compileJava FAILED
             ^
1 actionable task: 1 executed
1 error

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileJava'.
> Compilation failed; see the compiler error output for details.
```

Please make the necessary changes to ensure that the project build does not fail. It also looks like there are a few merge conflicts that need to be resolved.

------

Otherwise, thanks a bunch for the contribution! I like the variety of new hydration messages added. I will mark this pull request as hacktoberfest-accepted for now.

File: src/main/java/com/hydratereminder/dictionary/HydrateBreakMessageDictionary.java (hunk 1 above)
Comment (Zeesky-code) [reply src/main/java/com/hydratereminder/dictionary/HydrateBreakMessageDictionary.java]: Yeah, I've added it.

File: src/main/java/com/hydratereminder/dictionary/HydrateBreakMessageDictionary.java (hunk 2 above)
Comment (Zeesky-code) [reply src/main/java/com/hydratereminder/dictionary/HydrateBreakMessageDictionary.java]: Yeah, I've added it.

**Review by mslowiak — CHANGES_REQUESTED** [review]
Could you rename or squash your commits to make them more meaningful?
`please work` doesn't look like a good commit message ;)

**Review by jmakhack — APPROVED** [review]

**Comment by jmakhack** [comment]
@all-contributors please add @Zeesky-code for code

**Pull request merged**